projects/plugins/jetpack/modules/likes/queuehandler.js

```diff
@@ -290,6 +290,8 @@ function jetpackLoadLikeWidgetIframe( wrapperID ) {
 		postLikesFrame.width = '200px';
 		postLikesFrame.frameBorder = '0';
 		postLikesFrame.scrolling = 'no';
+		postLikesFrame.title = 'Like or Reblog';
```

Comment:
Can we add the title to the `iFrame` data attribute so that it can be translatable and use it here through `$wrapper.data( 'title' );` like in https://github.com/Automattic/jetpack/blob/e13487d0dbbe4c743739b6f1a3e37c143a8c2614/projects/plugins/jetpack/modules/likes/queuehandler.js#L287 ?

Reply:
Never mind, since we also add `aria-hidden` there is no point in translating the title.

Reply:
Shouldn't we translate it anyway, for folks using screen readers?

Reply:
`aria-hidden` hides it from screen readers. But as it seems it also hides child elements.

> To hide an element to screen readers (and child elements), simply add the aria-hidden="true" attribute.

Hiding `Like` or `Reblog` from screen readers is not desirable. In that case let's remove `aria-hidden` and let's make the `title` translatable.